samples/RaspberryPiInstructions.md

```diff
@@ -1,70 +1,129 @@
+
+Note: We've had the most success with Debian on ARM64 for the Pi.
 
-## Publishing an app to run on the Pi:
+For 64-bit, You need to install the following components using the following commands, here for Debian:
 
-* Run `dotnet publish -r <runtime identifier>` for example `dotnet publish -r win-arm` to publish the application for windows and `dotnet publish -r linux-arm` for Linux running on Raspberry Pi.
+```console
+sudo apt-get update
+sudo apt-get install libc6 libgcc1 libgssapi-krb5-2 libicu60 liblttng-ust0 libssl1.0.0 libstdc++6 zlib1g
+``` 
 
-* Under `./bin/Debug/netcoreapp2.1/<runtime identifier>/publish` or `.\bin\Debug\netcoreapp2.1\<runtime identifier>\publish` you will see the whole self contained app that you need to copy to your Raspberry Pi.
+Note: There are various tricks that makes it easier to manage installing Linux images:
 
+* [Install operating system images](https://www.raspberrypi.org/documentation/installation/installing-images/README.md)
+* [Start ssh automatically at boot time](https://raspberrypi.stackexchange.com/questions/1747/starting-ssh-automatically-at-boot-time).
 
-## Getting the app to run on the Pi.
+## Installing the .NET Core SDK on Rasberry Pi for Linux ARM32/ARM64
 
-### Linux
+The best way to install .NET Core on ARM platforms is currently via curl. We are working on other options.
 
-* Install [Linux](https://www.raspberrypi.org/downloads/) on your Pi.
+Note: The pattern documented below is the same one we use for [Docker](https://github.com/dotnet/dotnet-docker). You can look at .NET Core Dockerfiles to learn more about specific installation needs.
 
-* Install the [platform dependencies from your distro's package manager](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md) for .NET Core. .NET Core depends on some packages from the Linux package manager as prerequisites to running your application.
+1. Download .NET Core via curl (choose the version that works for you):
+  * 2.2 ARM32: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.2.103/dotnet-sdk-2.2.103-linux-arm.tar.gz`
+  * 3.0 ARM32: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/3.0.100-preview-010184/dotnet-sdk-3.0.100-preview-010184-linux-arm.tar.gz`
+  * 3.0 ARM64: `curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Sdk/3.0.100-preview-010184/dotnet-sdk-3.0.100-preview-010184-linux-arm64.tar.gz`
 
-For Raspbian [Debian 9 Jessie](https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore2x#install-net-core-for-debian-8-or-debian-9-64-bit) you need to do the following:
-```
-sudo apt-get update
-sudo apt-get install curl libunwind8 gettext apt-transport-https
+  Note: The latest download links are available at the [.NET Core Download page](https://dotnet.microsoft.com/download/archives).
+2. Unpack .NET Core to a global location:
```

Comment:
The formatting is not correct here.  When rendered on GitHub, '2.' is not on a separate line.